samples/hcls_2015.shex

```diff
@@ -144,12 +144,12 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
     !dct:accrualPeriodicity . ,
 #Availability/Distributions
     (dcat:distribution IRI)*, ###TODO restrict the shape of the IRI but could be hard to test in practice
-    ###dcat:landingPage IRI, ###MAY property
+    `may` dcat:landingPage IRI, ###MAY property
     !dct:format .,
-    ###dcat:accessURL IRI, ### is a MAY property
+    `may` dcat:accessURL IRI, ### is a MAY property
     !dcat:downloadURL .,
     !void:dataDump .,
-    !void:sparqlEndpoint ., ###SHOULD NOT being strict here
+    `should not` !void:sparqlEndpoint ., ###SHOULD NOT being strict here
```

Comment:
You should use `should` rather than `should not`. The not is implied by the ! in the shex rule.